src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -165,8 +172,11 @@ def get_lang_from_content(src_path, html=False):
         runtime_details_dict['file_loc'] = package_netcore_file
         runtime_details_dict['default_sku'] = 'F1'
     else:  # TODO: Update the doc when the detection logic gets updated
-        raise CLIError("Could not auto-detect the runtime stack of your app, "
-                       "see 'https://go.microsoft.com/fwlink/?linkid=2109470' for more information")
+        raise CLIError("Could not auto-detect the runtime stack of your app.\n"
+                       "HINT: Are you in the right folder?\n"
+                        "For more information, see "
+                        + Fore.BLUE + Style.BRIGHT + "'https://go.microsoft.com/fwlink/?linkid=2109470'"
```

Comment:
as shared in another comment, pls try to use existing CLI error framework for this.

Reply:
Changed back to regular